src/marqo/marqo_cloud_instance_mappings.py

```diff
@@ -1,37 +1,48 @@
 import time
-from marqo.marqo_logging import mq_logger
+from typing import Optional
+
 import requests
 from requests.exceptions import Timeout
+
 from marqo.errors import (
     MarqoCloudIndexNotFoundError,
     MarqoCloudIndexNotReadyError,
 )
+from marqo.instance_mappings import InstanceMappings
+from marqo.marqo_logging import mq_logger
+
 
+class MarqoCloudInstanceMappings(InstanceMappings):
+    _MARQO_CLOUD_URL = "https://api.marqo.ai"
 
-class MarqoUrlResolver:
     def __init__(self, api_key=None, expiration_time: int = 15):
```

Comment:
We will need more context, and perhaps a better name for the `expiration_time` parameter. It is not clear what this parameter does when we instantiate this class.

Reply:
@pandu-k chatGPT proposed url_cache_duration it sounds good to me